code/game/atoms_movable.dm

```diff
@@ -202,19 +202,19 @@
 		if(isturf(loc))
 			buckled_mob.glide_size = glide_size // Setting loc apparently does animate with glide size.
 			buckled_mob.forceMove(loc)
-			refresh_buckled_mob()
+			refresh_buckled_mob(0)
```

Comment:
This is a side-change that prevents atoms that a mob is buckled to from causing them to slide around via animate() when changing dir, specifically due to the janicart looking like absolute bollocks when moving.